yarn-project/aztec-sandbox/src/bin/index.ts

```diff
@@ -9,13 +10,27 @@ import { dirname, resolve } from 'path';
 
 import { setupFileDebugLog } from '../logging.js';
 import { createSandbox } from '../sandbox.js';
-import { startHttpRpcServer } from '../server.js';
 import { github, splash } from '../splash.js';
 
 const { SERVER_PORT = 8080 } = process.env;
 
 const logger = createDebugLogger('aztec:sandbox');
 
+/**
+ * Creates the sandbox from provided config and deploys any initial L1 and L2 contracts
+ */
+async function createAndDeploySandbox() {
```

Comment:
Not a fan of the name, since we're not "deploying the sandbox". But `createSandboxAndInitializeAccounts` is a bit long...

Reply:
Change it to `createAndInitialiseSandbox`. Hope that's a happy middle ground :-)